stustapay/core/schema/db/0010-payout-run.sql

```diff
@@ -0,0 +1,41 @@
+-- revision: c5c4ae30
+-- requires: dc0af35b
+
+
+create table if not exists payout_run (
+    id bigint primary key generated always as identity (start with 1),
+    created_by text not null unique,
+    created_at timestamptz not null
+);
+
+alter table customer_info add column payout_run_id bigint references payout_run(id) on delete cascade;
+alter table customer_info add column error text;
+alter table customer_info add column payout_export boolean default false not null;
```

Comment:
shouldn't the default be `true` so we can exclude problematic customers by setting it `false` for them?

Reply:
I thought it might be a good idea to set our current customers to `payout_export` `false` to avoid exporting all as we probably want to test with just a few exports in the beginning where we would set `payout_export` to `true` manually. And later we could set the rest to `true`. But as a default I agree, that `true` would be more sane.